### ISSUE TITLE
A sapper version would be nice too.

### ISSUE THREAD

**Issue by QuantumInformation** [babycourageous/netlify-identity-demo-svelte#1]

**Comment by babycourageous** [comment]
@QuantumInformation I totally agree! I'll start drafting one up and let ya know when it's up if you like.

**Comment by QuantumInformation** [comment]
Cool, I'm not sure how you would pull it off with a static export though which is my use case for netlify.

**Comment by QuantumInformation** [comment]
So I've been trying it with a sapper app and browser stuff is not available, so just to get things to work on sapper local dev I am trying stuff like:
```
function createUser() {
  const localUser = process.browser
    ? JSON.parse(localStorage.getItem('gotrue.user'))
    : undefined```

**Comment by QuantumInformation** [comment]
Do you mind if I make a mention of this repo in my youtube videos? btw do you have a contact email or smth?

**Comment by babycourageous** [comment]
No sweat at all! Thanks for asking!

I don't have anything really besides my personal email.

**Comment by WeAreELIC** [comment]
Any update on a Sapper version?

**Comment by babycourageous** [comment]
@WeAreELIC I actually had this on my backburner but I haven't had a chance unfortunately.

I haven't grabbed Sapper in a while. Maybe this is a good reason to jog my memory. 
Sorry I don't have more of an update for ya.

**Comment by quantuminformation** [comment]
its hard when your working to do open source, at least for me)
